### PR TITLE
Increase test coverage for svg.py

### PR DESCRIPTION
**Pull request by yiweelan**
Increase test coverage for syg.py.  #395  This relates to the lines not covered in "transform_to_rect_viewport" function. 

**Checklist**:

- [x] The GitHub pipeline is OK (green), 
      meaning that both `pylint` (static code analyzer) and `black` (code formatter) are happy with the changes of this PR.

- [ ] A unit test is covering the code added / modified by this PR

- [ ] This PR is ready to be merged 

- [ ] In case of a new feature, docstrings have been added, with also some documentation in the `docs/` folder

- [ ] A mention of the change is present in `CHANGELOG.md`

By submitting this pull request, I confirm that my contribution is made under the terms of the [GNU LGPL 3.0 license](https://github.com/PyFPDF/fpdf2/blob/master/LICENSE).

### CODE REVIEW

File: test/svg/test_transform_viewport.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+def test_rect_transform_width_exception():
+    svg_data = """<?xml version="1.0" standalone="no"?>
+    <svg width="10%" height="10cm" viewBox="0 0 1000 1000" xmlns="http://www.w3.org/2000/svg" version="1.1">
+    </svg>
+    """
+
+    svg = fpdf.svg.SVGObject(svg_data)
+    with pytest.raises(ValueError) as error:
+        svg.transform_to_rect_viewport(1, None, 10)
```
Comment (Lucas-C) [inline test/svg/test_transform_viewport.py:13]:
```suggestion
        svg.transform_to_rect_viewport(scale=1, width=None, height=10)
```

File: test/svg/test_transform_viewport.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+def test_rect_transform_height_exception():
+    svg_data = """<?xml version="1.0" standalone="no"?>
+    <svg width="10cm" height="10%" viewBox="0 0 1000 1000" xmlns="http://www.w3.org/2000/svg" version="1.1">
+    </svg>
+    """
+
+    svg = fpdf.svg.SVGObject(svg_data)
+    with pytest.raises(ValueError) as error:
+        svg.transform_to_rect_viewport(1, 10, None)
```
Comment (Lucas-C) [inline test/svg/test_transform_viewport.py:27]:
```suggestion
        svg.transform_to_rect_viewport(scale=1, width=10, height=None)
```

File: test/svg/test_transform_viewport.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+
+def test_rect_transform_vw_zero():
+    svg_data = """<?xml version="1.0" standalone="no"?>
+    <svg width="10cm" height="10cm" viewBox="0 0 0 1000" xmlns="http://www.w3.org/2000/svg" version="1.1">
+    </svg>
+    """
+
+    svg = fpdf.svg.SVGObject(svg_data)
+    width, height, context = svg.transform_to_rect_viewport(2, 10, 5)
```
Comment (Lucas-C) [inline test/svg/test_transform_viewport.py:40]:
```suggestion
    width, height, context = svg.transform_to_rect_viewport(scale=2, width=10, height=5)
```

**Review by Lucas-C — APPROVED** [review]

**Comment by Lucas-C** [comment]
Thank you @yiweelan for this PR!

@allcontributors please add @yiweelan for code

**Comment by Lucas-C** [comment]
@yiweelan are you OK with my minor code suggestions?

If so I'm going to merge them and then your pR

Just waiting for your feedback 😊

**Comment by yiweelan** [comment]
Yes, I am okay with the code suggestion. Sorry this is my first ever pull request, so I don't really know how everything works.

**Comment by Lucas-C** [comment]
No worries! Merged, thak you for your contribution :)

**Pull request merged**